Projects/Backendless.NET.Standard20/Backendless.NET.Standard20.csproj

```diff
@@ -6,7 +6,7 @@
     <PackageVersion>6.0.1.0</PackageVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
-    <DefineConstants>TRACE;DEBUG;NETSTANDARD2_0;PURE_CLIENT_LIB;WITHRT</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;NETSTANDARD2_0;PURE_CLIENT_LIB;WITHRT;UNITY_ANDROID;</DefineConstants>
```

Comment:
UNITY_ANDROID is a dynamic variable, it is injected by Unity when the app is running on Android

Reply:
fixed